pipeline_template.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version:
+
+date:
+
+pipelines:
+  - models:
+    model_name:
+    pipeline_name:
+    created_by:
+    creation_date:
+    version:
+    description:
+    job_name:
+    dataset:
+    metrics:
+    identifiable:
+    default:
+
+models:
+  - tasks:
+    api_call_function:
+    endpoint:
+    api_key:
+    inference_function:
+    content_type:
+    deployment_platform:
+    endpoint_name:
+    model_name:
+    created_by:
+    creation_date:
+    version:
+    description:
+    job_name:
+    dataset:
```

Comment:
This is the dataset for the model, and previously it was for the pipeline containing the models. It might be a bit repetitive and unnecessary for pipeline,